server/lib/db.js

```diff
@@ -60,13 +60,15 @@ async function initNedb(config) {
     connectionAccesses: getDatastore('connectionaccesses.db'),
     queries: getDatastore('queries.db'),
     queryHistory: getDatastore('queryhistory.db'),
+    serviceTokens: getDatastore('servicetokens.db'),
```

Comment:
I think this was missed with move to sequelize/sqlite

Reply:
fixed by 5096803f85d0d787b0db640c2df59dc0b4f3f5e2